frontend/app/src/services/defi/types.ts

```diff
@@ -26,19 +26,25 @@ export type SupportedDefiProtocols = typeof DEFI_PROTOCOLS[number];
 
 export interface ApiDSRMovement {
   readonly movement_type: DSRMovementType;
-  readonly gain_so_far: string;
-  readonly gain_so_far_usd_value: string;
-  readonly amount: string;
-  readonly amount_usd_value: string;
+  readonly gain_so_far: {
+    amount: string;
+    usd_value: string;
+  };
+  readonly value: {
```

Comment:
There is already a `Balance` type with the two properties, I will take care of it though.

Reply:
Ah all right, thanks. Ping me when it's done so that I can see where it was.